products/cockroachdb.md

```diff
@@ -145,11 +167,31 @@ releases:
 
 > [CockroachDB](http://cockroachdb.com/) is a distributed SQL database built on a transactional and
 > strongly-consistent key-value store, developed by Cockroach Labs.
+> {: .warning }
+> This page tracks the CockroachDB Self-Hosted releases - the Cloud and Serverless releases have their own
+> [support and upgrade policy](https://www.cockroachlabs.com/docs/cockroachcloud/upgrade-policy).
+> CockroachDB is released under a [staged delivery process](https://www.cockroachlabs.com/docs/releases/staged-release-process),
+> New releases are made available for CockroachDB Cloud clusters for two weeks before binaries are published for CockroachDB Self-Hosted downloads.
+> [CockroachDB Cloud](https://www.cockroachlabs.com/docs/releases/cloud) supports the latest major version of
+> CockroachDB and the version immediately preceding it.
```

Comment:
```suggestion
```

Should we really be this specific about products not documented on this page ? Isn't https://www.cockroachlabs.com/docs/cockroachcloud/upgrade-policy sufficient ?

Reply:
The two week delay felt important enough to mention, the rest felt like an easy win. But I did feel it was getting too verbose. I'll trim it to just the 2 week mention.